daq-2.0.6/os-daq-modules/daq_netvm.c

```diff
@@ -148,26 +163,38 @@ static int parse_args(char *inputstring, char **argv)
     return ap - argv;
 }
 
+/* need a sig handler for cleanup */
+void sig_handler(int sig) {
+    if (sig == SIGINT || sig == SIGTERM) {
+        //printf("Received user termination");
+        break_loop = 1;
+    }
+}
+
 /* Initialise the DAQ module and the DAQ context. */
 static int netvm_daq_initialize(const DAQ_Config_t *config, void **ctxt_ptr, char *errbuf, size_t errlen)
 {
     NetVM_Context_t *netvmc = NULL;
     DAQ_Dict *entry;
-    int ret, rval = DAQ_ERROR;
+    int arg_offset, rval = DAQ_ERROR;
     char *netvm_args = NULL;
     char argv0[] = "fake";
     char *argv[MAX_ARGS + 1];
     int argc;
 
-    //printf("->netvm_daq_initialize()\n");
+    /* these are needed for new standards */
+    struct onvm_nf_local_ctx *nf_local_ctx;
 
+    //printf("->netvm_daq_initialize()\n");
```

Comment:
Nit: Rm commented print statement. Or surround in a #if debug variable.

Reply:
Updated